src/_posts/2020-09-17-cassandra-usage-report-2020.markdown

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: "Cassandra Usage Report 2020"
```

Comment:
Have we considered using the term `Apache Cassandra` instead of `Cassandra` wherever applicable throughout this post?

Reply:
good catch. in the title, and its first usage on the page, needs to be "Apache Cassandra"